pints/_mcmc/__init__.py

```diff
@@ -363,7 +363,30 @@ def __init__(self, log_pdf, chains, x0, sigma0=None, method=None):
         self._max_iterations = None
         self.set_max_iterations()
 
-        # TODO: Add more stopping criteria
+        self.set_log_likelihood_storage(False)
+
+    def set_log_likelihood_storage(self, store=False):
+        """
+        Sets whether or not to store log-likelihood for each posterior sample
+        """
+        self._log_likelihood_storage = bool(store)
```

Comment:
We don't _only_ do Bayesian. Please have a look at the log pdf / log likelihood logging to disk

Reply:
@MichaelClerx Good point. I have now made two methods: log_likelihood_storage (for only Bayesian problems) and log_density_storage (for all problems). Think this is a good thing to do since certain users may want log_pdf whereas others want log_likelihoods. See what you think.